.github/workflows/zxcron-extended-test-suite.yaml

```diff
@@ -285,6 +285,31 @@ jobs:
         with:
           path: run.log
 
+      - name: Report failure (PagerDuty)
+        run: |
+          curl --request 'POST' \
+          --url 'https://events.pagerduty.com/v2/enqueue' \
+          --header 'Content-Type: application/json' \
+          --data '{
+            "payload": {
+              "summary": "Hedera Services - eXtended Test Suite Failure",
+              "severity": "error",
+              "source": "GitHub Actions - Extended Test Suite",
+              "custom_details": {
+                "Fetch XTS Candidate Tag": "${{ needs.fetch-xts-candidate.result }}",
+                "Execute eXtended Test Suite": "${{ needs.extended-test-suite.result }}",
+                "JRS Panel": "${{ needs.abbreviated-panel.result }}",
+                "Hedera Node JRS Panel": "${{ needs.hedera-node-jrs-panel.result }}",
+                "Tag as XTS-Passing": "${{ needs.tag-for-promotion.result }}",
+                "Source Commit": "<${{ github.server_url }}/${{ github.repository }}/commit/${{ needs.fetch-xts-candidate.outputs.xts-tag-commit }}>",
+                "Commit Author": "${{ needs.fetch-xts-candidate.outputs.xts-tag-commit-author }}",
+                "Workflow Run": "<${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}>"
+              }
+            },
+            "routing_key": "${{ secrets.PAGERDUTY_CITR_INTEGRATION_KEY }}",
```

Comment:
I don't see this key passed into the workflow. Does it need to be or is it an env token available implicitly to the whole workflow?

Reply:
It's a workflow dispatch trigger and a crib trigger. These have access to the repository secrets implicitly